runtime/picasso/Cargo.toml

```diff
@@ -37,6 +37,7 @@ balances = { package = "pallet-balances", git = "https://github.com/paritytech/s
 indices = { package = "pallet-indices", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12", default-features = false }
 randomness-collective-flip = { package = "pallet-randomness-collective-flip", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12", default-features = false }
 collective = { package = "pallet-collective", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12", default-features = false }
+identity = { package = "pallet-identity", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12", default-features = false }
```

Comment:
It is unusual to have this pattern even across parachain repositories. Auto generated benchmark code also does not play integrate well

Reply:
fuck. we are so bad.

Reply:
???

It is not bad or good. Just a fact

Reply:
i mean all 3rd party pallets should be exported without their weights?

Reply:
so we can measure these for our target hardware

Reply:
Maybe I wasn't specific enough.

It is not common to see `packages = { ... }` for all substrate/cumulus/polkadot dependencies and this feature makes auto generated code fail to compile because Parity assumed that imported dependencies have their canonical names. For example, I had to rename `frame_support` to `support` in the newly generated weights file of `Identity`.

Therefore, this situation is not related to pallets without weights or import of generated pallet weights

Reply:
oh, i see. weird renames. they hamper my copy pasted from other runtime. and there is another issue with weigths. :(